third_party/java/java_tools/README.md

```diff
@@ -21,4 +21,20 @@ third_party/java/java_tools/SingleJar_deploy.jar
 The following tools were built with bazel 0.21.0 at commit 019f13b64630ea7e7837a2ed8b664c4262322b1c by running:
 $ third_party/java/java_tools/update_java_tools.sh
 
-third_party/java/java_tools/jarjar_command_deploy.jar
\ No newline at end of file
+third_party/java/java_tools/jarjar_command_deploy.jar
```

Comment:
nit: Isn't this duplicated from the 0.22 (see line 38 in the same file?)

Reply:
The file didn't have a trailing newline, the only thing that changed on this line was trailing whitespace.

Reply:
Oh, also, I think the duplication with 38 is deliberate? The script is printing the list of files that change in each update, and this file changed in this update and the one below.